src/jvm/test/resources/expected_logs/validation_function_failure.txt

```diff
@@ -19,8 +19,9 @@ All operations above the horizontal line | ----- | happen before those below the
 java.lang.IllegalStateException: Validation works!
 	at org.jetbrains.kotlinx.lincheck_test.representation.ValidationFunctionCallTest.validateWithError(ValidationFunctionTests.kt:44)
```

Comment:
Should we shrink the stacktrace here? If so, let's make an issue for that.

Reply:
What do you mean by "shrinking" the stack trace?
Do you mean that we need to remove the lines like this one (like we do for exceptions thrown in actors?):
```
at org.jetbrains.kotlinx.lincheck.runner.FixedActiveThreadsExecutor.testThreadRunnable$lambda$10(FixedActiveThreadsExecutor.kt:172)
```

If so, then yes, there is a problem currently that validation function stack traces are handled differently compared to other stack traces printed by Lincheck.

Reply:
Here is the issue: https://github.com/JetBrains/lincheck/issues/438